sdk/src/test/scala/com/horizen/account/state/CertificateKeyRotationMsgProcessorTest.scala

```diff
@@ -80,12 +81,14 @@ class CertificateKeyRotationMsgProcessorTest
   }
 
   private def buildKeyRotationProof(`type`: KeyRotationProofType, index: Int, newKey: SchnorrSecret, oldSigningKey: SchnorrSecret, oldMasterKey: SchnorrSecret) = {
+    val messageToSign = SchnorrPublicKey.deserialize(newKey.publicImage().pubKeyBytes()).getHash.serializeFieldElement()
```

Comment:
memory leak here.
Need to "free" `SchnorrPublicKey`.
The same bug we have in other places (both account and utxo)
Let's move to utility function and be agnostic to cryptolib types.

Reply:
this one will be fixed in 0.6.0-snapshot3 and ported to this PR.